components/studio/habitat/plan.sh

```diff
@@ -13,6 +15,10 @@ pkg_build_deps=(core/coreutils
                 core/hab)
 pkg_bin_dirs=(bin)
 
+do_prepare() {
+  set_runtime_env "HAB_STUDIO_BACKLINE_PKG" "$(< $(pkg_path_for core/hab-backline)/IDENT)"
```

Comment:
Is this going to be a permanent thing, or is this basically some support code to bridge us to the glorious future when all this code is sane? 😅

Reply:
I suspect semi-permanent as `RUNTIME_ENVIRONMENT` is our interface for influencing the runtime (but not the internals!) of the studio. Though...  I just had a thought that may allow us to remove this. I'll run some experiments, but I think for now this should stay as-is.